projects/angular/src/layout/vertical-nav/_vertical-nav.clarity.scss

```
@@ -66,6 +66,7 @@
       border-width: vertical-nav-variables.$clr-vertical-nav-trigger-divider-border-width;
       border-style: solid;
       border-color: vertical-nav-variables.$clr-vertical-nav-trigger-divider-border-color;
+      border-top: 0; // to prevent double border - it uses div
```

Can you combine all the border styles into one?

```suggestion
      //Dims to prevent double border - it uses div
      border-top: vertical-nav-variables.$clr-vertical-nav-trigger-divider-border-width solid vertical-nav-variables.$clr-vertical-nav-trigger-divider-border-color;
```